performant_PoCs/focused-vision/focused_pipeline/main.py

```diff
@@ -0,0 +1,156 @@
+from pathlib import Path
+
+import depthai as dai
+from depthai_nodes.node import GatherData, ParsingNeuralNetwork, ImgDetectionsBridge
+from depthai_nodes.node.utils import generate_script_content
+
+from utils.annotation_node import AnnotationNode
+from utils.grid_eyes_annotation_node import GridEyesAnnotationNode
+from utils.arguments import initialize_argparser
+from utils.grid_layout_node import GridLayoutNode
+
+_, args = initialize_argparser()
+
+INPUT_WIDTH, INPUT_HEIGHT = 3840, 2160
+DET_MODEL = "luxonis/yunet:320x240"
```

Comment:
Both models used here are detection models, please find a better name for the yunet model, maybe in the context of this app, it could be named the `HEAD_DETECTION_MODEL`, the second one could be the `EYE_DETECTION_MODEL`